x/autopilot/keeper/liquidstake.go

```diff
@@ -72,12 +72,47 @@ func (k Keeper) RunLiquidStake(ctx sdk.Context, addr sdk.AccAddress, token sdk.C
 	}
 
 	msgServer := stakeibckeeper.NewMsgServerImpl(k.stakeibcKeeper)
-	_, err := msgServer.LiquidStake(
+	result, err := msgServer.LiquidStake(
 		sdk.WrapSDKContext(ctx),
 		msg,
 	)
 	if err != nil {
-		return errorsmod.Wrapf(sdkerrors.ErrInsufficientFunds, err.Error())
+		return errorsmod.Wrapf(err, err.Error())
 	}
-	return nil
+
+	if packetMetadata.IbcReceiver == "" {
+		return nil
+	}
+
+	hostZone, err := k.stakeibcKeeper.GetHostZoneFromHostDenom(ctx, token.Denom)
+	if err != nil {
+		return err
+	}
+
+	return k.IBCTransferStAsset(ctx, result.StToken, addr.String(), hostZone, packetMetadata)
+}
+
+func (k Keeper) IBCTransferStAsset(ctx sdk.Context, stAsset sdk.Coin, sender string, hostZone *stakeibctypes.HostZone, packetMetadata types.StakeibcPacketMetadata) error {
+	ibcTransferTimeoutNanos := k.stakeibcKeeper.GetParam(ctx, stakeibctypes.KeyIBCTransferTimeoutNanos)
+	timeoutTimestamp := uint64(ctx.BlockTime().UnixNano()) + ibcTransferTimeoutNanos
+	channelId := packetMetadata.TransferChannel
+	if channelId == "" {
+		channelId = hostZone.TransferChannelId
+	}
+	transferMsg := &transfertypes.MsgTransfer{
+		SourcePort:    transfertypes.PortID,
+		SourceChannel: channelId,
+		Token:         stAsset,
+		// TODO: does this reintroduce the bug in PFM where senders can be spoofed?
+		// If so, should we instead call PFM directly to forward the packet?
+		// Or should we obfuscate the sender, making it a random address?
+		Sender:           sender,
+		Receiver:         packetMetadata.IbcReceiver,
+		TimeoutTimestamp: timeoutTimestamp,
+		// TimeoutHeight:    clienttypes.Height{},
+		// Memo:             "stTokenIBCTransfer",
```

Comment:
nit: uncomment or remove

Reply:
nit: should we allow for a memo to be passed to the next chain? If this is technically challenging, absolutely don't think it needs to go into this version, but would allow for some more complex IBC-chaining (e.g. autopilot + swap on Osmosis)

Reply:
done

Reply:
> nit: should we allow for a memo to be passed to the next chain? If this is technically challenging, absolutely don't think it needs to go into this version, but would allow for some more complex IBC-chaining (e.g. autopilot + swap on Osmosis)

for now, we explicitly don't allow pfm / autopilot to interact. agree that it might be worth doing in the future, but I think it complicates the potential code paths we'd have to think through, so would vote to keep it simple for now!